### DISCUSSION TITLE
What exactly are these inputs?

### DISCUSSION THREAD

**Discussion by mainrs** [hlorenzi/customasm#154] [Q&A] [answered]
https://github.com/hlorenzi/customasm/blob/5de7a21d374ca8bb472d29441c2c986f9860334e/examples/nes/cpu6502.asm#L98-L107

I cannot figure out what `<{...}` and `#{...}` do. Maybe I am missing it but they do not seem to be documented

**Comment by hlorenzi** [comment, marked as answer]
Those are just generic tokens -- everything outside of `{ }` is. We could have chosen to use any other kind of punctuation there.

As for what they mean in the context of 6502 assembly, `#` marks an immediate-mode argument, and `<` is seen on some assembly flavors to mark exclusive zero-page addressing. (To be fair, in those flavors, `<` might actually just be an operator that takes the lowest 8 bits of a value, akin to ``` `8```)
